cache/README.md

```diff
@@ -20,8 +20,6 @@ install, and start **deis/cache**.
   daemon (default: *4001*)
 * **ETCD_PATH** sets the etcd directory where the cache announces its
   configuration (default: */deis/cache*)
-* **ETCD_TTL** sets the time-to-live before etcd purges a configuration
```

Comment:
not after the bin/boot to a go-based approach. the TTL is already hardcoded to 20 seconds: https://github.com/deis/deis/blob/master/cache/main.go#L18-L19